client/src/state/reducer.js

```diff
@@ -26,6 +32,21 @@ export default (
         error: action.payload,
         loading: false
       };
+    case POST_BEER:
+      return {
```

Comment:
Oops I made a mistake when I was setting this up the first time.

We don't want to wipe out the previous state every time we get an action coming through the reducer so on every object we return we should add a `...state,` at the top of the object so that the previous state is merged in.

For example:

```js
case POST_BEER:
  return {
    ...state,
    error: undefined,
    loading: true
  };
```

You'll need to update all the return statements, even the ones that existed before your change.

Reply:
Fixed. Thanks.